### ISSUE TITLE
Consider dropping execa

### ISSUE THREAD

**Issue by frontsideair** [frontsideair/yarnhook#51]
The project can have zero dependencies without execa, which has listed their [benefits](https://github.com/sindresorhus/execa#why) but maybe we don't need any of them? Test extensively.

**Comment by devinrhode2** [comment]
Just my 2 cents, probably better to just keep execa, I don't think having
zero deps is all that interesting. Claiming only 1 dependency is pretty
good too

**Comment by frontsideair** [comment]
Agreed that it's not an urgency. But I gave the benefits a quick read and it doesn't seem like we would be missing out on anything if we dropped execa or cross-spawn. I need to do an investigation to be sure as I wouldn't want to leave Windows users behind for any reason.
